tiledb/sm/serialization/array_schema.cc

```diff
@@ -668,6 +668,55 @@ shared_ptr<Domain> domain_from_capnp(
   return make_shared<Domain>(HERE(), cell_order, dims, tile_order);
 }
 
+void dimension_label_to_capnp(
+    const DimensionLabel& dimension_label,
+    capnp::DimensionLabel::Builder* dim_label_builder,
+    const bool client_side) {
+  dim_label_builder->setDimensionId(dimension_label.dimension_index());
+  dim_label_builder->setName(dimension_label.name());
+  dim_label_builder->setUri(dimension_label.uri().to_string());
```

Comment:
The code comment should be clarified. That is referring to the URI stored in the dimension label schema, not the URI used by the parent array to locate the dimension label.

Reply:
Also is this relative?

Reply:
It can be (usually is). That is set on line 683.

Reply:
Fixed the comment.
About relative vs absolute. My understanding is that currently all dimension label URIs are relative, and that absolute if for future use (for groups? I don't remember, @jp-dark can confirm). Since I don't have an example of an absolute URI I don't know how exactly to truncate it to relative. I usually do that by using the helper methods : `serialize_array_uri_to_relative` and `deserialize_array_uri_to_absolute` , but I am not sure how to update them for dimension labels case.
I added a check that throws if the URI is absolute, so that in future use cases where absolute URIs are used, this serialization path will have to get implemented.

Reply:
That above is correct. Currently, all dimension label URIs are relative. The absolute path is for allowing external dimension labels (not yet implemented).